README.md

```diff
@@ -11,15 +11,60 @@ https://hub.docker.com/r/stshriv/kube-agent/
 
-2. ci.sh handles calling get_version.py in a vsts release definition. Provide ci.sh with the same parameters as get_version.
+### version.sh
+*version.sh* handles collecting the build IDs from a specified build definition so we can tag our images using them. version.sh handles calling into get_version.py.
 
-3. render.sh is used to create a dockerfile for each version returned by get_version.py. dockerfile.template provides a baseline dockerfile for building images with available msftkube wheels.
+To run version.sh use a Shell++ task with the filepath set to ci/version.sh
+
+Parameters:
+
+| Short Command |    Full Command    |  Type  | Required |   Default Value   |           Description           |
+|---------------|--------------------|--------|----------|-------------------|---------------------------------|
+|      -a       | --account          | string | required |                   | VSTS account name               |
+|      -b       | --branch           | string |          |'refs/heads/master'| VSTS Build Branch               |
+|      -c       | --count            |  int   |          |        0          | The number of builds to return  |
+|      -d       | --definition       |  int   | required |                   | The build definition ID         |
+|      -e       | --environmentOutput| string |          |                   | Script output                   |
+|      -p       | --project          | string | required |                   | VSTS Project within the account |
+|      -s       | --buildstatus      | string |          |                   | build status filter             |
+|      -t       | --token            | string |          |     NoneType      | User token to access account    |
+|      -u       | --user             | string |          |     NoneType      | Username associated with token  |
+|      -v       | --version          | float  |          |       4.1         | VSTS REST API version           |
+|      -V       | --verbose          | boolean|          |      False        | Print verbose output            |
+
+### render.sh
+*render.sh* handles the creation of version specific dockerfiles by substituting placeholder text within dockerfile.template.
+
+Use Shell++ script with the filepath set to ci/render.sh and provide the following parameters.
+1. Versions (file): A file containing the versions from version.sh. This file is specified with the -e parameter above.
+2. Output (filename): render will append to a file the full filepath of all dockerfiles generated after performing template substitution. Each line will contain `<docker file location> <build id/version>` for use in the build step of continuous integration.
+3. The placeholder text to substitute. 
+
+### build.sh
+*build.sh* handles docker build and push commands and can take up to 4 arguments.
```

Comment:
mention here or above what the tags will actually look like. just kube-agent:<msftkube-version> or is something else appended.  Do we want to publish a parallel tag with the git commit/build id of this kube-agent repo appended as well? 

One of my new goals is to get msftkube to release a latest that people actually consume so all the git repos aren't stuck on old versions.

Reply:
I'll update the readme next. Merging this to make sure our image builds can move forward.